### PR TITLE
change the mk file

### PR DESCRIPTION
**Pull request by jxhgzs**
1. Failed to run android project by cocos2d-2.1beta3-x-2.1.1
   error report:
         Fatal connect socket fail in socket_abstract_client

### CODE REVIEW

File: cocos2dx/Android.mk [hunk 1]
```diff
@@ -139,7 +139,8 @@ touch_dispatcher/CCTouch.cpp
 LOCAL_EXPORT_C_INCLUDES := $(LOCAL_PATH) \
                     $(LOCAL_PATH)/include \
                     $(LOCAL_PATH)/kazmath/include \
-                    $(LOCAL_PATH)/platform/android
+                    $(LOCAL_PATH)/platform/android \
+                    $(LOCAL_PATH)/platform/textures
```
Comment (dumganhar) [inline cocos2dx/Android.mk:143]: Where is the folder `$(LOCAL_PATH)/platform/textures`?

File: CocosDenshion/android/Android.mk [hunk 2]
```diff
@@ -16,7 +16,8 @@ LOCAL_C_INCLUDES := $(LOCAL_PATH)/../include \
                     $(LOCAL_PATH)/../../cocos2dx \
                     $(LOCAL_PATH)/../../cocos2dx/include \
                     $(LOCAL_PATH)/../../cocos2dx/kazmath/include \
-                    $(LOCAL_PATH)/../../cocos2dx/platform/android
+                    $(LOCAL_PATH)/../../cocos2dx/platform/android \
+					$(LOCAL_PATH)/../../cocos2dx/textures
```
Comment (dumganhar) [inline CocosDenshion/android/Android.mk:20]: Please use relative path when including header files like this:
`#include "textures/CCTexture2D.h"`

File: cocos2dx/Android.mk (hunk 1 above)
Comment (jxhgzs) [inline cocos2dx/Android.mk:143]: cocos2d-x/cocos2dx/textures

File: CocosDenshion/android/Android.mk (hunk 2 above)
Comment (jxhgzs) [inline CocosDenshion/android/Android.mk:20]: this is cocos2dx problem;

**Comment by jxhgzs** [comment]
file is: CCTMXXMLParser.h
build cocos2dx android error is:  

Compile++ thumb  : cocos2dx_static <= cocos2d.cpp
In file included from /Users/zhangcheng/Desktop/CocoSpark/CocoSpark/Dependences/cocos2d-x/cocos2dx/tilemap_parallax_nodes/CCTMXLayer.h:32:0,
                 from /Users/zhangcheng/Desktop/CocoSpark/CocoSpark/Dependences/cocos2d-x/cocos2dx/include/cocos2d.h:232,
                 from /Users/zhangcheng/Desktop/CocoSpark/CocoSpark/Dependences/cocos2d-x/cocos2dx/cocos2d.cpp:27:
/Users/zhangcheng/Desktop/CocoSpark/CocoSpark/Dependences/cocos2d-x/cocos2dx/tilemap_parallax_nodes/CCTMXXMLParser.h:35:25: fatal error: CCTexture2D.h: No such file or directory

**Comment by jxhgzs** [comment]
the same error in: cocos2d-x/CocosDenshion/android/Android.mk

**Pull request closed without merging**